src/layouts/AllServices.tsx

```diff
@@ -0,0 +1,542 @@
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/openshift/insights/vulnerability/">
+                          Vulnerability
+                        </Text>
+                      </Text>
+
+                      <Text component={TextVariants.p} className="subtitle">
+                        RHEL
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/insights/advisor/recommendations">
+                          Advisor
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/insights/compliance/">
+                          Compliance
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/edge/fleet-managements">
+                          Edge
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/insights/malware/">
+                          Malware
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/insights/patch/advisories">
+                          Patch
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/edge/cves">
+                          Vulnerability
+                        </Text>
+                      </Text>
+                    </TextContent>
+                  </CardBody>
+                </Card>
+                <Card isPlain>
+                  <CardTitle>
+                    <FontAwesomeIcon icon={faCreditCard} />
+                    Spend Management
+                  </CardTitle>
+                  <CardBody>
+                    <TextContent>
+                      <Text component={TextVariants.p}>Control costs and monitor committed spend.</Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/openshift/cost-management">
+                          Cost Management
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="#">
+                          Hybrid Committed Spend
+                        </Text>
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.p} href="">
```

Comment:
This should navigate the user to `/insights/subscriptions/inventory`

Reply:
@epwinchell Yup, PM Dan Oz needs to get back to us on this.